src/pudl/transform/eia860.py

```diff
@@ -1286,7 +1286,7 @@ def _core_eia860__cooling_equipment(
 
 
 @asset_check(asset=_core_eia860__cooling_equipment, blocking=True)
-def cooling_equipment_null_cols(cooling_equipment):  # pragma: no cover
+def cooling_equipment_null_cols(cooling_equipment):
```

Comment:
These are left over from when we were having trouble getting the asset checks to run during the integration tests.